config/action.d/pf.conf

```diff
@@ -4,16 +4,52 @@
 #
 # Author: Nick Hilliard <nick@foobar.org>
 # Modified by: Alexander Koeppe making PF work seamless and with IPv4 and IPv6
+# Further modified by: Zane C. Bowers-Hadley <vvelox@vvelox.net> to make it kill states for a IP so the
+#                      ban takes affect as well as not handling multiple protocols.
+
+# WARNING WARNING WARNING WARNING WARNING
+# WARNING WARNING WARNING WARNING WARNING
+# WARNING WARNING WARNING WARNING WARNING
+#
+# This will kill any state for a banned IP. This will ensure the ban actually takes affect.
+# This is need as pf has both the strenght and weakness that changes won't affect current
+# states till it is forced to, but unfortunately the only way to make that take affect is
+# to kil them.
+#
+# If some one wishes to fix that, this will requiring writing a tool to grab the session
```

Comment:
Hammering all states for that IP instead of just those for a specific port and IP.

Wrote that forgetting about awk off hand. Yeah, definitely fixable and able to hit a specific one.